rpcclient.go

```diff
@@ -1,17 +1,6 @@
 package sdk
 
-import "github.com/valyala/gorpc"
-
-// RPCClient is a client to manage all rpc calls
+//RPCClient is a client to manage all rpc calls
 type RPCClient interface {
-	Call(request interface{}) (response interface{}, err error)
-}
-
-func newRPC(address string) RPCClient {
-	rpc := &gorpc.Client{
-		Addr: address, // "rpc.server.addr:12345",
-	}
-	rpc.Start()
-
-	return rpc
+	Call(result interface{}, method string, args ...interface{}) error
```

Comment:
so this interface will look like:
```
Call(request ShhRequest) (response interface{}, err error)
```
which will allow you to use `geth rpc` on the pinger like:
```
... Call(request *ShhRequest) (response interface{}, err error) {
	c.rpc.Call(&response, request.Method, request.Args...)
```